src/app/[navSlug]/[[...tabName]]/NavPageSpace.tsx

```
@@ -0,0 +1,140 @@
+  // Initialize frame ready state for mini app support
+  useEffect(() => {
+    if (!isFrameReady) setFrameReady();
+  }, [isFrameReady, setFrameReady]);
+
+  // If space is not found, show not found component
+  if (isNotFound) {
+    return <SpaceNotFound />;
+  }
+
+  // If we found a pending space but spaceData doesn't have the spaceId, update it
+  const effectiveSpaceData = useMemo(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)